src/handlers/channel.ts

```diff
@@ -427,13 +428,16 @@ export async function editChannelOverwrite(
   overwrites: Overwrite[],
   reason?: string,
 ) {
-  const gid = cache.channels.get(channelID)!.guildID;
+  const guildID = cache.channels.get(channelID)!.guildID;
```

Comment:
```suggestion
  const guildID = cache.channels.get(channelID)?.guildID;
```
Much more type safety